Nodejs/Product/Nodejs/NodejsPackage.cs

```diff
@@ -661,6 +661,21 @@ internal sealed partial class NodejsPackage : CommonPackage {
             LogLooseFileAnalysisLevel();
         }
 
+        private VsProjectAnalyzer CreateLooseVsProjectAnalyzer() {
```

Comment:
Are there any non-repl flows through `GetAnalyzer` that could be regressed by changing `DefaultAnalyzer`?